### PR TITLE
(maint) AIO path updates

### PR DESCRIPTION
**Pull request by joshcooper**
Fix up commented out code and documentation referring to old AIO
locations.

### CODE REVIEW

File: website/configure/client.md [hunk 1]
```diff
@@ -129,8 +129,8 @@ The `mco` client is usually run interactively from an admin workstation, but can
...
-* `/etc/mcollective/client.cfg` --- A global client config file, which will be used if `~/.mcollective` and `/etc/puppetlabs/agent/mcollective/client.cfg` don't exist. If you are using this file, read access to it and the external credentials it refers to should be strictly controlled, probably limited to the root user.
+* `/etc/puppetlabs/mcollective/client.cfg` --- A global client config file, which will be used (by versions >= 2.8.0) if `~/.mcollective` doesn't exist. If you are using this file, read access to it and the external credentials it refers to should be strictly controlled, probably limited to the root user.
+* `/etc/mcollective/client.cfg` --- A global client config file, which will be used if `~/.mcollective` and `/etc/puppetlabs/mcollective/client.cfg` don't exist. If you are using this file, read access to it and the external credentials it refers to should be strictly controlled, probably limited to the root user.
```
Comment (joshcooper) [inline website/configure/client.md]: Given AIO, will mcollective look for a global client config file in `/etc/mcollective/client.cfg`?

Comment (richardc) [inline website/configure/client.md]: It looks in both locations, AIO first.

**Comment by richardc** [comment]
Looks good but can we retarget this to 2.8.x.  Documentation changed in master is auto-published, so updating documentation is something we do on the 2.8.x branch and would have been something I'd have picked up as part of the 2.8.1 release tickets.

File: website/configure/client.md (hunk 1 above)
Comment (richardc) [inline website/configure/client.md]: Urgh, just realised this is now not strictly correct and might need even more caveating.

2.8.0 uses first of ~/.mcollective /etc/puppetlabs/agent/mcollective/client.cfg /etc/mcollective/client.cfg

2.8.1 will use first of ~/.mcollective /etc/puppetlabs/mcollective/client.cfg /etc/mcollective/client.cfg

**Pull request closed without merging**